airbyte-integrations/connectors/source-chargebee/acceptance-test-config.yml

```diff
@@ -12,7 +12,7 @@ tests:
   discovery:
     - config_path: "secrets/config.json"
       backward_compatibility_tests_config:
-        disable_for_version: "0.1.12"
+        disable_for_version: "0.2.0"
```

Comment:
can you clarify in the code why we are disabling backward compatibility checks? why is backwards compatibility failing?

Reply:
I was in a hurry and accidentally published low-code version 0.2.0, there are some breaking changes in schemas

Reply:
is there a reason we can't put this in the low-code migration PR assuming it gets reviewed today?

Reply:
Ok, then I'll  close this PR and move everything inside [low-code migration PR](https://github.com/airbytehq/airbyte/pull/21688)